site/content/docs/manifest/environment.en.md

```diff
@@ -159,6 +159,33 @@ List of [public AWS Certificate Manager certificate](https://docs.aws.amazon.com
 By attaching public certificates to your load balancer, you can associate your Load Balanced Web Services with a domain name and reach them with HTTPS. 
 See the [Developing/Domains](../developing/domain.en.md#use-domain-in-your-existing-validated-certificates) guide to learn more about how to redeploy services using [`http.alias`](./lb-web-service.en.md#http-alias).
 
+<span class="parent-field">http.public.</span><a id="http-public-access-logs" href="#http-public-access-logs" class="field">`access_logs`</a> <span class="type">Boolean or Map</span>   
+Enable Elastic Load Balancing access logs.   
```

Comment:
very nit 🥺 (there's a few of these in here)
```suggestion
Enable Elastic Load Balancing access logs.
```

Reply:
also maybe we should add a link to https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html

Reply:
We should leave the trailing spaces so that markdown adds a newline char 😄

Reply:
it does add it without trailing spaces 🤔

Reply:
whenever I press enter for the next line it doesn't really add it. So I add trailing spaces which works fine.